### PR TITLE
TokenAccess added

### PR DESCRIPTION
**Pull request by e7hdk**
Created access token

### CODE REVIEW

File: project/backend/app/register/views.py [hunk 1]
```diff
@@ -97,11 +99,14 @@ def register(request):
 
         send_mail(user, request)
 
+        skill1 = SkillLevel.objects.create(level_name=level1)
+        skill2 = SkillLevel.objects.create(level_name=level2)
```
Comment (GozdeUnver) [inline project/backend/app/register/views.py:103]: You shouldn't create these skill levels, instead you should get them from the post/models.py skillevel models

File: project/backend/app/register/views.py [hunk 2]
```diff
@@ -138,12 +143,34 @@ def login_user(request):
             return JsonResponse(context)
 
         login(request, user)
+        postjson = {
+            'username': username,
+            'password': password
+        }
+        token = requests.post("http://127.0.0.1:8000/api/token/", json=postjson)
```
Comment (GozdeUnver) [inline project/backend/app/register/views.py:150]: I think there is something wron here, you are sending a request to your local address

Comment (e7hdk) [reply project/backend/app/register/views.py:150]: oh got it, I will change it quickly

**Review by GozdeUnver — APPROVED** [review]
they seem okay but it should be tested on amazon and frontend requests

**Pull request merged**